pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -4603,16 +4599,18 @@ private void internalSetReplicatedSubscriptionStatusForNonPartitionedTopic(Async
                 }
 
                 ((PersistentTopic) topic).checkReplicatedSubscriptionControllerState();
-                log.info("[{}] Changed replicated subscription status to {} - {} {}", clientAppId(), enabled, topicName,
+                log.info("[{}] Changed replicated subscription status to {} - {} {}", clientAppId(), enabled,
+                        topicName,
                         subName);
                 asyncResponse.resume(Response.noContent().build());
             } else {
                 asyncResponse.resume(new RestException(Status.METHOD_NOT_ALLOWED,
                         "Cannot enable/disable replicated subscriptions on non-persistent topics"));
             }
-        } catch (Exception e) {
+        }).exceptionally(e -> {
             resumeAsyncResponseExceptionally(asyncResponse, e);
```

Comment:
- Maybe we need ``e.getCause`` here.
- We need log some information when get exception.